src/tests/Bubble.test.tsx

```
@@ -0,0 +1,45 @@
+import { render, screen } from '@testing-library/react';
```

ESLint

[error] 'screen' is defined but never used.

(@typescript-eslint/no-unused-vars)